app/controllers/api/account.php

```diff
@@ -3167,6 +3349,9 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
             if ($oldTarget instanceof Document && !$oldTarget->isEmpty()) {
                 Authorization::skip(fn () => $dbForProject->updateDocument('targets', $oldTarget->getId(), $oldTarget->setAttribute('identifier', $phone)));
             }
+
+            sendPhoneChangeEmail($locale, $user, $project, $queueForMails, $oldPhone, $phone);
```

Comment:
Let's update tests to ensure all those emails arrive after respective actions, and include expected contents (for example phone number, and a specific subject)